portugues/revisa/tenta-consegue/ti-cruel/desenvolve-app/angrycocos/angrycocos.md

```diff
@@ -0,0 +1 @@
+Você cria uma estratégia inicial de publicar seu app, Angry Cocos, nos app stores de mercados marginais. Quando o app chega ao seu milésimo download, você percebe que o nome não faz muito sentido.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.